Epam.TestAutomationProject/Epam.TestAutomation.Tests.BDD/Steps/HeaderSteps.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Epam.TestAutomation.Core.Browser;
+using Epam.TestAutomation.Core.Elements;
+using Epam.TestAutomation.Core.Utils;
+using Epam.TestAutomation.Pages.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium.Interactions;
+using TechTalk.SpecFlow;
+
+namespace Epam.TestAutomation.Tests.BDD.Steps
+{
+    [Binding]
+    public class HeaderSteps
+    {
+        private MainPage _mainPage = new();
+        private JobListingsPage _jobListingsPage = new();
+        private Actions _actions = new Actions(Browser.Instance.Driver);
```

Comment:
it is better to create a Helper class for actions and do not initialize it in steps